modules/gapi/include/opencv2/gapi/infer/ie.hpp

```diff
@@ -137,6 +142,12 @@ template<typename Net> class Params {
         return *this;
     }
 
+    Params& setNumRequests(size_t nireq) {
```

Comment:
other methods start with `.cfg`

Reply:
... `pluginConfig` is some fatal inconsistency, though.

Reply:
Fixed